src/test/java/htsjdk/utils/SamtoolsTestUtils.java

```diff
@@ -0,0 +1,91 @@
+package htsjdk.utils;
+
+import htsjdk.samtools.util.*;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+/**
+ * Test utilities for running samtools from htsjdk tests.
+ */
+public class SamtoolsTestUtils {
+    private static final Log LOG = Log.getInstance(SamtoolsTestUtils.class);
+    private static final String SAMTOOLS_BIN_PROPERTY = "HTSJDK_SAMTOOLS_BIN";
```

Comment:
a bit confusing since it's named property but it's actually checked as an environment variable

Reply:
Yeah fixed.